### DISCUSSION TITLE
How to connect to the SQLite DB?

### DISCUSSION THREAD

**Discussion by jmschneider815** [f1db/f1db#18] [Q&A] [answered]
Hi dumb question here: can someone direct me to instructions to how to connect to this SQLite DB? I'd like to query it from R or PowerBI

**Comment by marceloverdijk** [comment, marked as answer]
Hi @jmschneider815  

For R I think this will help: https://cran.r-project.org/web/packages/RSQLite/vignettes/RSQLite.html
And for PowerBI see https://community.fabric.microsoft.com/t5/Power-Query/connecting-a-SQLite-database/td-p/224205

Cheers,
Marcel

**Comment by jmschneider815** [comment]
Thanks for the quick reply! Excited to dive into this data!
